include/i2c.h

```diff
@@ -55,6 +56,24 @@ extern "C" {
 /** Controller to act as Master. */
 #define I2C_MODE_MASTER			(1 << 4)
 
+static inline int __i2c_map_dt_bitrate(u32_t bitrate)
```

Comment:
The dt-bindings headers should try and be standalone (so avoid Zephyr specific defines or code there).  I can move this into a private drivers/i2c/i2c-priv.h header